velox/dwio/dwrf/test/TestOrcColumnStatistics.cpp

```diff
@@ -0,0 +1,175 @@
+  testStringEmptyStats(
+      columnStatisticsWrapper, (void*)&proto, DwrfFormat::kOrc);
+}
+
+TEST(StatisticsBuilder, stringLengthThreshold) {
+  testStringLengthThreshold();
+}
+
+TEST(StatisticsBuilder, stringLengthOverflow) {
+  proto::orc::ColumnStatistics proto;
+  proto.set_numberofvalues(1);
+  auto strProto = proto.mutable_stringstatistics();
+  auto columnStatisticsWrapper = ColumnStatisticsWrapper(&proto);
+  testStringLengthOverflow(columnStatisticsWrapper, strProto, DwrfFormat::kOrc);
+}
+
+TEST(StatisticsBuilder, boolean) {
+  testBoolean();
+}
+
+TEST(StatisticsBuilder, booleanMissingStats) {
+  proto::orc::ColumnStatistics proto;
+  auto boolProto = proto.mutable_bucketstatistics();
+  auto columnStatisticsWrapper = ColumnStatisticsWrapper(&proto);
+  testBooleanMissingStats(columnStatisticsWrapper, boolProto, DwrfFormat::kOrc);
+}
+
+TEST(StatisticsBuilder, booleanEmptyStats) {
+  proto::orc::ColumnStatistics proto;
+  proto.set_numberofvalues(0);
+  auto columnStatisticsWrapper = ColumnStatisticsWrapper(&proto);
+  testBooleanEmptyStats(
+      columnStatisticsWrapper, (void*)&proto, DwrfFormat::kOrc);
+}
+
+TEST(StatisticsBuilder, basic) {
+  testBasic();
+}
+
+TEST(StatisticsBuilder, basicMissingStats) {
+  proto::orc::ColumnStatistics proto;
+  auto columnStatisticsWrapper = ColumnStatisticsWrapper(&proto);
+  testBasicMissingStats(columnStatisticsWrapper);
+}
+
+TEST(StatisticsBuilder, basicHasNull) {
+  proto::orc::ColumnStatistics proto;
+  auto columnStatisticsWrapper = ColumnStatisticsWrapper(&proto);
+  testBasicHasNull(columnStatisticsWrapper, DwrfFormat::kOrc);
+}
+
+TEST(StatisticsBuilder, binary) {
+  testBinary();
+}
+
+TEST(StatisticsBuilder, binaryMissingStats) {
+  proto::orc::ColumnStatistics proto;
+  auto binProto = proto.mutable_binarystatistics();
+  auto columnStatisticsWrapper = ColumnStatisticsWrapper(&proto);
+  testBinaryMissingStats(columnStatisticsWrapper, binProto, DwrfFormat::kOrc);
+}
+
+TEST(StatisticsBuilder, binaryEmptyStats) {
+  proto::orc::ColumnStatistics proto;
+  proto.set_numberofvalues(0);
+  auto columnStatisticsWrapper = ColumnStatisticsWrapper(&proto);
+  testBinaryEmptyStats(
+      columnStatisticsWrapper, (void*)&proto, DwrfFormat::kOrc);
+}
+
+TEST(StatisticsBuilder, binaryLengthOverflow) {
+  proto::orc::ColumnStatistics proto;
+  auto binProto = proto.mutable_binarystatistics();
+  auto columnStatisticsWrapper = ColumnStatisticsWrapper(&proto);
+  testBinaryLengthOverflow(columnStatisticsWrapper, binProto, DwrfFormat::kOrc);
+}
+
+TEST(StatisticsBuilder, initialSize) {
+  testInitialSize();
+}
+
+TEST(MapStatistics, orcUnsupportedMapStatistics) {
+  proto::orc::ColumnStatistics proto;
+  auto columnStatisticsWrapper = ColumnStatisticsWrapper(&proto);
+  ASSERT_FALSE(columnStatisticsWrapper.hasMapStatistics());
+  ASSERT_THROW(
+      columnStatisticsWrapper.mapStatistics(),
+      ::facebook::velox::VeloxRuntimeError);
+}
```

Comment:
since the Linter didn't run on this file it missed adding a new line at the end of a file, could you add one?

Reply:
A newline has been added to the end of the file, thank you.